plugins/bigquery/dbt/include/bigquery/macros/adapters.sql

```diff
@@ -128,6 +128,37 @@
   {% do adapter.rename_relation(from_relation, to_relation) %}
 {% endmacro %}
 
+{% macro alter_relation_add_remove_columns(relation, add_columns = none, remove_columns = none) -%}
+  {{ return(adapter.dispatch('alter_relation_add_remove_columns')(relation, add_columns, remove_columns)) }}
+{% endmacro %}
```

Comment:
You can remove this dispatch call and just keep the `bigquery__` implementation below